core/modules/file/file.install

```diff
@@ -713,3 +857,179 @@ function file_update_1002() {
+  $config->set('file_upload_wizard_skip_fields', update_variable_get('file_entity_file_upload_wizard_skip_fields', FALSE));
+  $config->set('protect_repeated_render', update_variable_get('file_entity_protect_repeated_render', TRUE));
+  $config->save();
+
+  // Remove converted variables.
+  update_variable_del('file_entity_max_filesize');
+  update_variable_del('file_entity_default_allowed_extensions');
+  update_variable_del('file_entity_default_file_directory');
+  update_variable_del('file_entity_allow_insecure_download');
+  update_variable_del('file_entity_file_upload_wizard_skip_file_type');
+  update_variable_del('file_entity_file_upload_wizard_skip_scheme');
+  update_variable_del('file_entity_file_upload_wizard_skip_fields');
+  update_variable_del('file_entity_protect_repeated_render');
+
+  // Delete unused variables.
+  update_variable_del('file_entity_alt');
+  update_variable_del('file_entity_title');
+}
+
+/**
+ * Add the default file types, or convert from Drupal.
+ */
+function file_update_1005() {
+  $default_file_types = array(
+    'audio' => array(
+      'type' => 'audio',
+      'name' => 'Audio',
+      'description' => 'An Audio file is a sound recording.',
+      'mimetypes' => array(
+        'audio/*'
+      ),
+      'disabled' => FALSE,
+      'storage' => 4,
+      'module' => 'file'
+    ),
+    'document' => array(
+      'type' => 'document',
+      'name' => 'Document',
+      'description' => 'A document file is written information.',
+      'mimetypes' => array(
+          'text/plain',
+          'application/msword',
+          'application/vnd.ms-excel',
+          'application/pdf',
+          'application/vnd.ms-powerpoint',
+          'application/vnd.oasis.opendocument.text',
+          'application/vnd.oasis.opendocument.spreadsheet',
+          'application/vnd.oasis.opendocument.presentation',
+          'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+          'application/vnd.openxmlformats-officedocument.presentationml.presentation',
+          'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+      ),
+      'disabled' => FALSE,
+      'storage' => 4,
+      'module' => 'file',
+    ),
+    'image' => array(
+      'type' => 'image',
+      'name' => 'Image',
+      'description' => 'An image file is a still visual.',
+      'mimetypes' => array(
+          'image/*',
+      ),
+      'disabled' => FALSE,
+      'storage' => 4,
+      'module' => 'file',
+    ),
+    'video' => array(
+      'type' => 'video',
+      'name' => 'Video',
+      'description' => 'A Video file is a moving visual recording.',
+      'mimetypes' => array(
+          'video/*',
+      ),
+      'disabled' => FALSE,
+      'storage' => 4,
+      'module' => 'file',
+    ),
+  );
+  foreach ($default_file_types as $machine => $type) {
+    if (!file_type_get_type($machine)) {
```

Comment:
Instead of using an API function which may change in the future, or in some weird situations the .module file might not be loaded, use `$config = config('file.type.' . $machine);` and then check for `$config->isNew()`.

Reply:
done, though and API function shouldn't change.